SyntaxError: Unexpected end of JSON input

Hello, I am using this package for my discord bot which uses Discord.js v13. Upon using it I get this error:

`[ERROR] SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at IncomingMessage.<anonymous> (C:\Users\User\Desktop\Durchie v13\node_modules\urban\lib\urban.js:83:24)
    at IncomingMessage.emit (node:events:406:35)
    at endReadableNT (node:internal/streams/readable:1331:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)`
    
    Any idea why?

--- a/lib/urban.js
+++ b/lib/urban.js
@@ -1,4 +1,4 @@
-var http = require('http'),
+var https = require('https'),
     util = require('util'),
     EventEmitter = require('events').EventEmitter,
     randomWord;
@@ -69,8 +69,7 @@ Dictionary.fn.getData = function(page) {
   var _path = (!randomWord) ? '/v0/define?page=' + self.page + '&term=' + self.words : '/v0/random';
   if (this._req && this._ended === 0) this._req.abort(); // abort if there's already one request happening
   this._ended = false;
-  this._req = http.get({
-    port: 80,
+  this._req = https.get({
     host: 'api.urbandictionary.com',
     path: _path
   }, function(res) {

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "author": "Murilo Santana <mvrilo@gmail.com>",
   "name": "urban",
   "description": "Simple API and command line for Urban Dictionary",
-  "version": "0.3.2",
+  "version": "0.3.3",
   "license": "MIT",
   "main": "./lib/urban.js",
   "repository": {

